src/classes/Dovetail/DovetailApi.php

```diff
@@ -145,21 +145,28 @@ public function rest_get_episode( \WP_REST_Request $request ) {
 			];
 		}
 
-		return $this->rest_response( $return, $resp );
+		return $this->rest_response(
+			$return,
+			$resp,
+			[
+				'Cache-Control' => 'no-cache, no-store, must-revalidate, max-age=0, s-max-age=0',
```

Comment:
Can't hurt, just in case some weird CDN is looking for 1 directive but not the others.